int16[] vels
int16[] gains
---
uint16[] strainvals
uint16[] fingerpositions
